marklogic-unit-test-modules/src/main/ml-modules/root/test/test-helper.mjs

```diff
@@ -0,0 +1,20 @@
+
+const helperExportFunctions = {};
+
+const proxy = new Proxy(helperExportFunctions, {
```

Comment:
I think this needs a comment on it to explain what `proxy` refers to - from looking at `test.mjs`, I can figure out (I think) that it's allowing each of the functions in `test-helper.xqy` to be referenced. If that's correct, I recommend both adding a comment to explain why a user may want to use this helper, and also rename `proxy` to `testHelperProxy` or something a little more self-describing so it's more clear what it's proxying. 

I'm also wondering - how essential is this for meeting the goal of #150, which is to allow for .mjs test modules to be found and executed? This strikes me as solving a different problem - i.e. make it easier for .mjs (and .sjs?) test writers to reuse the functions in test-helper.xqy. If so, let's do that in a separate PR so it's easy to tell what problem is solved by each PR.

Reply:
I'll add a comment and rename. 

I think it should go in with this PR, though, as it is used to test the other changes since mjs code can't import XQuery without this sort of proxy/inbetween code.